LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2024 Alphabet Inc.
```

Comment:
This is not needed I don't think.

Reply:
Ok, I dropped that commit and rebased off of master, so it's just the package.json change now